test/integration/default/build/aws.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = "~> 0.10.0"
```

Comment:
This seems to be an unrelated change.  Terraform changes a lot on minor version bumps; our pinning to v0.10.x is intentional, to avoid breakage when testers use incompatible versions.  If you have a different verision of terraform, you might consider using tfenv, which allows you to switch versions easily.

If you feel this is a needed change, please open a separate issue/PR.

Reply:
Fixed and added pinning to the AWS provider so the two linking play well together.